s3proxy/cmd/main.go

```diff
@@ -96,6 +100,7 @@ func parseFlags() (cmdFlags, error) {
 	region := flag.String("region", defaultRegion, "AWS region in which target bucket is located")
 	certLocation := flag.String("cert", defaultCertLocation, "location of TLS certificate")
 	kmsEndpoint := flag.String("kms", "key-service.kube-system:9000", "endpoint of the KMS service to get key encryption keys from")
+	forwardMultipartReqs := flag.Bool("allow-multipart", false, "forward multipart requests to the target bucket; beware: this may leak data to AWS")
```

Comment:
```suggestion
	forwardMultipartReqs := flag.Bool("allow-multipart", false, "forward multipart requests to the target bucket; beware: this may store unencrypted data on AWS")
```
Should we be more explicit here?
Maybe also "see the docs for more info"?

Reply:
Added both. Not sure about formatting.
I feel like your suggestion is as explicit as we can get. Apart from capital letters. But I feel like we are mentioning the aspect every step of the way now (docs, help text, logs).